src/Command/Metrics/MetricsCommandBase.php

```diff
@@ -36,12 +36,12 @@ abstract class MetricsCommandBase extends CommandBase
     private $fields = [
         // Grid.
         'local' => [
-            'cpu_used' => "SUM((`cpu.user` + `cpu.kernel`) / `interval`, 'service')",
-            'cpu_percent' => "SUM(100 * (`cpu.user` + `cpu.kernel`) / (`interval` * `cpu.cores`), 'service')",
-            'cpu_limit' => "AVG(`cpu.cores`, 'service')",
+            'cpu_used' => "AVG(SUM((`cpu.user` + `cpu.kernel`) / `interval`, 'service', 'instance'), 'service')",
```

Comment:
why do you do AVG here? would it not be enough to just project the instance as well as the service?

Reply:
This is the same as Console's `cpu.average.value` query. For this PR I want to fix what we are displaying which is one value per service, not one per instance; later we could add some kind of option to split up the instances. This is consistent with the display for Dedicated envs.